src/App.js

```diff
@@ -44,6 +46,8 @@ function App() {
                 <td><img src={contact.pictureUrl} alt={contact.name}/></td>
                 <td>{contact.name}</td>
                 <td>{contact.popularity}</td>
+                <td>{ contact.wonOscar === true ? <span className='trophee'>🏆</span> : ""}</td>
+                <td>{contact.wonEmmy === true ? <span className='trophee'>🏆</span> : "" }</td>
```

Comment:
bellow two lines are equivalent, as they both return true. 

`if (contact.wonOscar === true) `
`if (contact.wonOscar)`

Remember that inside if condition (or in this case in ternary expression condition), we get true or false.

So when we say:
` if ('hello') `-->it will return true

Below gives falsy:
`if ('')` -> returns false
`if (false) `-> retruns false

https://developer.mozilla.org/en-US/docs/Glossary/Falsy